### PR TITLE
Improvements around setting GDPR consent

### PR DESCRIPTION
**Pull request by MaratGilyazov**
- make gdpr_consent optional parameter in init() method 
- separate set_consent method

### CODE REVIEW

File: ironsource/api/ironsource.script_api [hunk 1]
```diff
@@ -15,6 +15,15 @@
       - name: gdpr_consent
```
Comment (alexeyfeskov) [inline ironsource/api/ironsource.script_api:15]: This parameter will be ignored and may mislead the user

File: example/ui.gui_script [hunk 2]
```diff
@@ -122,7 +122,8 @@ function init(self)
   
   if ironsource then
     log(self, "ironsource init with app_key: " .. YOUR_APP_KEY)
-    ironsource.init(YOUR_APP_KEY, GDPR_CONSENT)
+    ironsource.set_consent(GDPR_CONSENT)
```
Comment (alexeyfeskov) [inline example/ui.gui_script]: [Instruction](https://developers.ironsrc.com/ironsource-mobile/android/advanced-settings/#step-1) says:
`It’s recommended to set the API prior to SDK Initialization.`
I missed this earlier!
May be add comment in example?

**Review by alexeyfeskov — CHANGES_REQUESTED** [review]
There are only minor notes:
- README.md contains wrong version of used SDK-s (both Android ain iOS)
- [autocomplete](https://github.com/MaratGilyazov/def_ironsource/pull/4/files#r389243187)
- [comment lua](https://github.com/MaratGilyazov/def_ironsource/pull/4/files#r389242783)

Everything else looks good. Thanks!

File: ironsource/api/ironsource.script_api (hunk 1 above)
Comment (MaratGilyazov) [reply ironsource/api/ironsource.script_api:15]: Actually it is optional and if set, then set_consent() will be invoked before init().
I've added "optional: true" to this parameter in .script_api

File: example/ui.gui_script (hunk 2 above)
Comment (MaratGilyazov) [reply example/ui.gui_script]: I've removed the whole set_consent() call from the example, it's not really needed here.

**Pull request merged**